server/src/app.ts

```diff
@@ -14,8 +15,10 @@ app.set('view engine', 'ejs');
 app.set('views', path.join(__dirname, 'views'));
 
 app.use(express.json());
-
 app.use('/api', getRoutes());
+if (!isDevelopment) {
+  app.use(express.static(path.join(__dirname, '../client')));
+}
```

Comment:
serves build file in /client

Reply:
so we all need to update our local NODE_ENV?

Reply:
ooo ok got it